lib/customer-deposit-wallet/rest/Cardano/Wallet/Deposit/REST/Start.hs

```diff
@@ -42,20 +90,64 @@ loadDepositWalletFromDisk
     -> WalletResource
     -> IO ()
 loadDepositWalletFromDisk tr dir env resource = do
-    result <- flip runWalletResourceM resource $ do
-        test <- walletExists dir
-        when test $ do
-            lg tr "Loading wallet from" dir
-            loadWallet env dir
-            lg tr "Wallet loaded from" dir
+    result <- runExceptT $ do
+        exists <- ExceptT $ flip runWalletResourceM resource $ do
+            test <- walletExists dir
+            liftIO $ print test
+            when test $ do
+                lg tr "Loading wallet from" dir
+                loadWallet env dir
+                lg tr "Wallet loaded from" dir
+            pure test
+        liftIO $ threadDelay 1_000_000
+        when exists $ do
+            ExceptT $ mockFundTheWallet (networkEnv env) resource
+            ExceptT $ flip runWalletResourceM resource $ do
+                    liftIO $ putStrLn "Available balance"
+                    availableBalance >>= liftIO . print
+                    liftIO $ putStrLn "Tx history by customer"
+                    getTxHistoryByCustomer >>= liftIO . print
+                    liftIO $ putStrLn "Tx history by time"
+                    getTxHistoryByTime >>= liftIO . print
+                    liftIO $ putStrLn "List customers"
+                    listCustomers >>= liftIO . print
+                    liftIO $ putStrLn "UTxO"
     case result of
         Left e -> error $ show e
         Right _ -> pure ()
 
-fakeBootEnv :: MonadIO m => WalletBootEnv m
-fakeBootEnv =
-    ( WalletBootEnv
+mockFundTheWallet
+    :: NetworkEnv IO z
+    -> WalletResource
+    -> IO (Either ErrWalletResource ())
+mockFundTheWallet network resource = do
+    resolveSlot <- slotToUTCTime network
+    flip runWalletResourceM resource $ do
+        onWalletInstance $ \w ->
+            onWalletState w
+                $ Delta.update
+                $ Delta.Replace . f resolveSlot
```

Comment:
```suggestion
```

Uhh, instead of adding a block to the wallet state directly — could you please use `postTx` from the `NetworkEnv`? The implementation of this function in `newNetworkEnvMock` will take care of forging a new block. 🤓 

The idea is to create a transaction using `Write` that sends `1_000_000` ADA to the desired customer address. You can take the money out of thin air, the mock `NetworkEnv` will accept that.